### PR TITLE
Docs-update

### PR DESCRIPTION
**Pull request by prakhargupta1**
Data binding and deployment updates. Things to fix:
1. Deployment icon (done)
2. Verify deployment content (done)

Screenshot 2023-03-29 at 1 27 31 PM

Screenshot 2023-03-29 at 1 27 14 PM

### CODE REVIEW

**Comment by prakhargupta1** [comment]
It would be great, if we can add a thin border on all of our images. Something like core docs also has:

Screenshot 2023-03-29 at 1 38 44 PM

Or if there something better that someone might want to suggest.

File: docs/data/toolpad/data-binding.md [hunk 1]
```diff
@@ -37,3 +37,15 @@ Empty field:
 Field with value:
 
 <img src="/static/toolpad/docs/data-binding/bind-6.png" alt="Binding result 2" width="902px" />
+
+## Event handling
+
+Components like MUI Button have onClick event handler props. This can be binded to run a piece of code to mutate the page state.
```
Comment (apedroferreira) [inline docs/data/toolpad/data-binding.md:43]:
```suggestion
Some components, such as `Button`, have event handler props. Event handler props can be binded to run a piece of code to mutate the page state.
```

File: docs/data/toolpad/data-binding.md [hunk 2]
```diff
@@ -37,3 +37,15 @@ Empty field:
...
+
+(In the example below you can see how on Click of a button, the value is copied from one component to the other)
```
Comment (apedroferreira) [inline docs/data/toolpad/data-binding.md:45]:
```suggestion
(In the example below you can see how after clicking the button, the value is copied from one component to the other)
```

File: docs/data/toolpad/deployment.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Deployment
+
+<p class="description">Deploying a Toolpad app is a 3-step process and quite like any other react application.</p>
```
Comment (apedroferreira) [inline docs/data/toolpad/deployment.md:3]:
```suggestion
<p class="description">Deploying a Toolpad app is a 3-step process and quite like any other React application.</p>
```

File: docs/data/toolpad/deployment.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+npm start
+```
+
+This command will create an optimized production build for the Toolpad app and will output what files it has generated and how large each file is:
+
+```sh
+npm run build
+```
+
+Once the build has been made. You can deploy it to any service of your choice!
```
Comment (apedroferreira) [inline docs/data/toolpad/deployment.md:17]:
```suggestion
Once the build has been made, you can deploy it to any service of your choice!
```

File: docs/data/toolpad/deployment.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+<p class="description">Deploying a Toolpad app is a 3-step process and quite like any other react application.</p>
+
+To serve your app locally:
+
+```sh
+npm start
+```
+
+This command will create an optimized production build for the Toolpad app and will output what files it has generated and how large each file is:
```
Comment (apedroferreira) [inline docs/data/toolpad/deployment.md:11]:
```suggestion
This command will create an optimized production build for the Toolpad app and will output the generated files:
```

File: docs/public/static/toolpad/docs/data-binding/bind-11.png
Comment (apedroferreira) [inline docs/public/static/toolpad/docs/data-binding/bind-11.png:1]: This image seems a bit small. 
Maybe we can have clearer "before" and "after" images so we can see how the binding changed things?

**Comment by apedroferreira** [comment]
Deployment icon: maybe a rocket? :)

**Comment by apedroferreira** [comment]
> It would be great, if we can add a thin border on all of our images. Something like core docs also has:
> 
> Screenshot 2023-03-29 at 1 38 44 PM
> 
> Or if there something better that someone might want to suggest.

I think whatever Core does on their images we could do the same, since the docs look similar.
Let me know if I should look into it!

**Review by apedroferreira — CHANGES_REQUESTED** [review]
Nice one, Prakhar, just left some feedback and suggestions for copy changes!
All my opinion of course so feel free to disagree with any of them.

**Comment by apedroferreira** [comment]
I've added a commit document `toolpad-display=standalone` since we will add it soon and there is a `Deployment` section in this PR where it feels appropriate.
I didn't include any images as maybe we should have a nice looking app to take screenshots from, and it didn't feel completely necessary in this case.

File: docs/data/toolpad/data-binding.md (hunk 1 above)
Comment (bharatkashyap) [reply docs/data/toolpad/data-binding.md:43]:
```suggestion
Some components have event handler props, such as the `Button` component and its `onClick` prop. Event handler props can be bound to run a piece of code to mutate the page state.
```

File: docs/data/toolpad/deployment.md (hunk 3 above)
Comment (bharatkashyap) [reply docs/data/toolpad/deployment.md:3]:
```suggestion
<p class="description">Deploying a Toolpad app is like deploying any other React application and has 3 steps:</p>
```

File: docs/public/static/toolpad/docs/data-binding/bind-11.png
Comment (prakhargupta1) [reply docs/public/static/toolpad/docs/data-binding/bind-11.png:1]: In the pic above this one, I have tried to show the binding and in the background there is the before image. Hopefully, it should be ok.

**Review by apedroferreira — APPROVED** [review]

**Comment by prakhargupta1** [comment]
> Deployment icon: maybe a rocket? :)

That should be good! @apedroferreira Need your help in this. As per pages.ts it has something to do with mono repo docs.

**Comment by apedroferreira** [comment]
> > Deployment icon: maybe a rocket? :)
> 
> That should be good! @apedroferreira Need your help in this. As per pages.ts it has something to do with mono repo docs.

Just changing the icon? I'll try it.

**Comment by prakhargupta1** [comment]
> Just changing the icon? I'll try it.

Yes

**Comment by apedroferreira** [comment]
> > Just changing the icon? I'll try it.
> 
> Yes

My bad, looks like it's only possible to use icons from this file in those docs nav items https://github.com/mui/material-ui/blob/master/docs/src/modules/components/AppNavDrawerItem.js
So we can stick to the code one for now, but I can make a PR there to add the rocket icon, I guess I'll make a PR.

**Pull request merged**